spec/requests/code_owner_projects.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Code Owner Projects' do
+  describe '#index' do
+    let(:user) { create(:user) }
+    subject { get "/users/#{user.id}/projects" }
+    context 'when a user has not projects as code owner' do
+      before do
+        subject
+      end
+
+      it_behaves_like 'controller index response'
+
+      it 'returns no projects' do
+        expect(assigns(:projects)).to be_empty
+      end
+    end
+
+    context 'when projects has projects as code owner' do
```

Comment:
"projects has projects"? 😂

Reply:
JAJAJAJAJAJ the copy pasting doing its work